schemas/org.cinnamon.desktop.keybindings.gschema.xml.in

```diff
@@ -29,6 +29,11 @@
       <summary>Magnifier zoom out</summary>
       <description>Binding for the magnifier to zoom out</description>
     </key>
+    <key name="locate-pointer" type="s">
+      <default>'Control_L'</default>
```

Comment:
@JosephMcc Can you change this instead to:
```
<key name="locate-pointer" type="as">
  <default>['&lt;Control_L&gt;']</default>
```

or

```
<key name="locate-pointer" type="as">
  <default>['<Control_L>']</default>
```
(either one should work)

This would allow for multiple values to be set (such as both Ctrl keys or alternate keystrokes).